public/setup/atomicupdate/updatePath.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+$sql[]="UPDATE `{$dbprefix}acces` SET `page` = '/absence' WHERE `nom`='Absences - Index';";
```

Comment:
WHERE page = <name-of-the-page>. It's better since many routes can have the same name. It's not necessary the case here, but it's a good practice to update a column according to the same column